### PR TITLE
cripts: rename get/set macros to Get/Set

### PR DESCRIPTION
**Pull request by cmcfarlen**
The lowercase macros interfere with boringssl headers when used in the build.

### CODE REVIEW

File: include/cripts/Context.hpp [hunk 1]
```diff
@@ -100,8 +100,8 @@ class Context
 } // namespace Cript
 
 // This may be weird, but oh well for now.
-#define get()               _get(context)
-#define set(_value)         _set(context, _value)
+#define Get()               _get(context)
```
Comment (maskit) [inline include/cripts/Context.hpp:103]: Can we do this?
```c
#define Response::get()    Response::_get(context)
```

Comment (zwoop) [reply include/cripts/Context.hpp:103]: No, C++ doesn't allow this (I tried that long ago, it's a good idea, but no bueno).

```
/tmp/foo.cc:3:17: warning: ISO C99 requires whitespace after the macro name [-Wc99-extensions]
#define Response::get() Response::_get(context)
                ^
1 warning generated.
```

**Review by zwoop — APPROVED** [review]
This is good. For consistency, we agreed to change all public define / methods like this to Pascal case, rather than camel case. Consistency is important, particularly for this type of "scripting" API.

I'll take care of the rest from here.

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x

**Pull request merged**